### PR TITLE
Fix/network module typos

### PR DESCRIPTION
**Pull request by robphoenix**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

network/ios_*
##### ANSIBLE VERSION

```
ansible 2.1.0.0
```
##### SUMMARY

Fixes typos in network/ios_\* modules

### CODE REVIEW

File: network/ios/ios_config.py [hunk 1]
```diff
@@ -100,7 +100,7 @@
         against the contents of source.  There are times when it is not
         desirable to have the task get the current running-config for
         every task in a playbook.  The I(config) argument allows the
-        implementer to pass in the configuruation to use as the base
+        implementer to pass in the configuration to use as the base
         config for comparision.
```
Comment (dagwieers) [inline network/ios/ios_config.py]: You missed one here ;-)

Comment (robphoenix) [inline network/ios/ios_config.py]: ah tsk! wood for the trees an' all that, thanks for the catch. Will add it in.

**Comment by robphoenix** [comment]
I think I managed to coax git into updating that properly??

**Comment by dagwieers** [comment]
Looking good.

BTW It was just a drive-by correction, I only looked at the diff and noticed this one. Didn't proofread the whole documentation...

**Comment by gregdek** [comment]
Thanks @bordeltabernacle. To the current maintainers, @privateip please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks @bordeltabernacle for this PR. Unfortunately, it is not mergeable in its current state due to merge conflicts. Please rebase your PR. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

For help on how to do this cleanly please see http://docs.ansible.com/ansible/community.html#contributing-code-features-or-bugfixes

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@bordeltabernacle A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by robphoenix** [comment]
ready_for_review
@privateip I realise you're doing a lot of work on these modules at the moment, would it be better if I close this PR and maybe revisit it after the 2.2 release, when perhaps things will have settled down?
It is only typo corrections! I feel a bit like a teacher with a red pen! :smile:

**Comment by gregdek** [comment]
Thanks @bordeltabernacle. To the current maintainers, @privateip please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gundalow** [comment]
Thanks for this

shipit

**Comment by gregdek** [comment]
Thanks again to @bordeltabernacle for this PR, and thanks @privateip, @gundalow for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gundalow** [comment]
Merged, thanks!

**Comment by robphoenix** [comment]
:+1:  :tada: Thanks!

**Pull request merged**